libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -76,6 +115,10 @@ public override string GetVersion()
             return StringUtils.Hash(sb.ToString());
         }
 
+        /// <summary>
+        /// Enumerate child dialog dependencies so they can be added to the containers dialog set.
+        /// </summary>
+        /// <returns>dialog enumeration.</returns>
```

Comment:
First uppercase letter
```suggestion
        /// <returns>Dialog enumeration.</returns>
```